### PR TITLE
Improve CLI e2e go tests

### PR DESCRIPTION
**Pull request by praveen4g0**
# Changes

I have Added E2E cli tests for pipelines alone. 
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by praveen4g0** [comment]
cc @piyush-garg @vdemeester @hrishin  @sthaha

**Comment by chmouel** [comment]
IMHO we need to figure out about the flakyness of go-netflix before merging this, see #499 for details.

/hold

**Comment by praveen4g0** [comment]
@vdemeester  do you see still see any blocker to merge this pr??

**Comment by vdemeester** [comment]
>     1. Do we know how much of the old suite of tests is covered in this conversion/ how similar the tests are? I'll start doing a more thorough review of this as well.

I'm temporarly adding back the old ones to make sure we don't loose coverage

>     2. I get that this is a first attempt to get all the e2e tests over to go, but the organization of the tests themselves is hard to follow. `pipeline_e2e_test.go` should be broken up into different files, but that's perhaps something we need to iterate on.

Yep, I plan to follow-up to clean a little bit of stuff and split those tests and organize them a little bit better :wink:

File: go.mod [hunk 1]
```diff
@@ -26,9 +25,12 @@ require (
 	github.com/tektoncd/pipeline v0.10.1
 	github.com/tektoncd/plumbing v0.0.0-20191218171343-56a836c50336
 	github.com/tektoncd/triggers v0.2.1-preview.0.20200130181252-68f571334722
+	go.opencensus.io v0.22.1
 	go.uber.org/multierr v1.1.0
 	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898
```
Comment (ppitonak) [inline go.mod]: https://godoc.org/golang.org/x/xerrors

> These functions were incorporated into the standard library's errors package in Go 1.13

Comment (vdemeester) [reply go.mod]: @ppitonak indeed (with a slight signature change). Anyway, we can get rid of those in a follow-up (also, depending on our dependency it might become a transitive dep anway)
